mlflow/utils/environment.py

```diff
@@ -20,6 +20,11 @@
 _CONSTRAINTS_FILE_NAME = "constraints.txt"
 
 
+class EnvManager(Enum):
+    LOCAL = "local"
+    CONDA = "conda"
+
```

Comment:
```suggestion
    CONDA = "conda"
    VIRTUALENV = "virtualenv"

```

`VIRTUALENV` will be added in https://github.com/mlflow/mlflow/pull/5380